packages/core/src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -485,7 +485,12 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
                 if (this._isCancellationError(result.error)) {
                     throw new PaymentMethodCancelledError();
                 }
-                throw new Error(result.error.message);
+
+                if (result.error.code === 'payment_intent_authentication_failure') {
+                  throw new Error('User did not authenticate');
+                } else {
+                    throw new Error(result.error.message);
+                }
```

Comment:
For this change, please refer to #1286 and #1367 and their sibling PRs in order to translate your custom message.

```suggestion
                if (this._isAuthError()) {
                    throw new StripeV3Error(StripeV3ErrorType.AuthFailure);
                }

                throw new Error(result.error.message);
```